router/http/src/main/scala/io/buoyant/router/http/ForwardClientCertFilter.scala

```diff
@@ -0,0 +1,66 @@
+package io.buoyant.router.http
+
+import java.security.MessageDigest
```

Comment:
linkerd import style is to have a single import block (no blank lines between imports) all alphabetically sorted together